x/subscription/keeper/subscription.go

```diff
@@ -687,7 +687,7 @@ func (k Keeper) DelProjectFromSubscription(ctx sdk.Context, consumer, name strin
 func (k Keeper) delAllProjectsFromSubscription(ctx sdk.Context, consumer string) {
 	allProjectsIDs := k.projectsKeeper.GetAllProjectsForSubscription(ctx, consumer)
 	for _, projectID := range allProjectsIDs {
-		err := k.projectsKeeper.DeleteProject(ctx, consumer, projectID)
+		err := k.projectsKeeper.PurgeProject(ctx, projectID)
```

Comment:
what happens on an error? this is called in begin block

Reply:
on the next line below you can see that it's simply printed. Should I change anything regarding this?